src/insertWord.js

```diff
@@ -23,7 +22,7 @@ async function addWord() {
     return false;
   }
 
-  const word = insertUi.createWordForUser(JSON.parse(localStorage.getItem(LOCALSTORAGE_KEY)));
+  const word = insertUi.createWordForUser(JSON.parse(localStorage.getItem(CURR_USER_KEY)));
```

Comment:
Did you import `CURR_USER_KEY` from `Constant.js`? I don't see it anywhere in this file.